ko/sql/catalog.rst

```diff
@@ -531,23 +534,23 @@ _db_index_key
 
 인덱스에 대한 키 정보이며 index_of에 대한 인덱스가 생성되어 있다.
 
-+-------------------+--------------+--------------------------------------------------+
-| 속성명            | 데이터 타입  | 설명                                             |
-+===================+==============+==================================================+
-| index_of          | _db_index    | 키 속성이 속하는 인덱스                          |
-+-------------------+--------------+--------------------------------------------------+
-| key_attr_name     | VARCHAR(255) | 키를 구성하는 속성명                             |
-+-------------------+--------------+--------------------------------------------------+
-| key_order         | INTEGER      | 키에서 속성이 위치한 순서로 0부터 시작함         |
-+-------------------+--------------+--------------------------------------------------+
-| asc_desc          | INTEGER      | 속성 값의 순서가 내림차순이면 1, 그렇지 않으면 0 |
-+-------------------+--------------+--------------------------------------------------+
-| key_prefix_length | INTEGER      | 키로 사용할 prefix의 길이                        |
-+-------------------+--------------+--------------------------------------------------+
-| func              | VARCHAR(255) | 함수 기반 인덱스의 함수 표현식                   |
-+-------------------+--------------+--------------------------------------------------+
-| status            | INTEGER      | 인덱스 상태                                      |
-+-------------------+--------------+--------------------------------------------------+
++-------------------+---------------+--------------------------------------------------+
+| 속성명            | 데이터 타입   | 설명                                             |
++===================+===============+==================================================+
+| index_of          | _db_index     | 키 속성이 속하는 인덱스                          |
++-------------------+---------------+--------------------------------------------------+
+| key_attr_name     | VARCHAR(255)  | 키를 구성하는 속성명                             |
++-------------------+---------------+--------------------------------------------------+
+| key_order         | INTEGER       | 키에서 속성이 위치한 순서로 0부터 시작함         |
++-------------------+---------------+--------------------------------------------------+
+| asc_desc          | INTEGER       | 속성 값의 순서가 내림차순이면 1, 그렇지 않으면 0 |
++-------------------+---------------+--------------------------------------------------+
+| key_prefix_length | INTEGER       | 키로 사용할 prefix의 길이                        |
++-------------------+---------------+--------------------------------------------------+
+| func              | VARCHAR(1023) | 함수 기반 인덱스의 함수 표현식                   |
```

Comment:
Why is the length of func increased?
Is this also related to User Schema?

Reply:
It relates to the [CBRD-24288](http://jira.cubrid.org/browse/CBRD-24288) issue.